project_liberation/main_page/templates/main_page/footer.haml

```diff
@@ -1,2 +1,51 @@
-%br
-  footer
\ No newline at end of file
+.footer.footer_color
+    .container
+        .row
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_logo
+                    %img{:src => "/static/main_page/images/footer/code_poets_logo_footer_white.png", :width => "208px", :height => "68px"}
```

Comment:
You mean `width` and `height`? Or `src` as well?
Figured since this is an image, I could get away with this here and just declare the image in it's desired output resolution and then add any extra attributes in the .sass.

Reply:
I meant `styles` so `width` and `height`

Reply:
Done